react-time-track/src/views/users.js

```diff
@@ -38,14 +39,18 @@ function Users() {
                     position: "relative",
                     font: "bold italic 45px/1.5 Helvetica, Verdana, sans-serif",
                     marginBottom: "20px"
-                  }}>
+                  }} key={usr.id}>
                   <Card css={{"p": {padding: "0"}}}>
                     <Li>Name: {usr.name}</Li>
                     <Li>Role: {usr.role}</Li>  
+                    <Link to={`/edit-user/${usr.id}`}>
+                      <FaEdit/>
+                    </Link>
                     </Card>                                                              
                   </li>                            
           );
         })}
+        {console.log(user)}
```

Comment:
What do you think about removing this console?

Reply:
I deleted it. thanks for notice it.